Makefile

```diff
@@ -95,7 +98,7 @@ release-linux: clean
 	cd native && RUSTFLAGS="-Ctarget-cpu=native -Ctarget-feature=-prefer-256-bit" cargo build --release
 	./mvnw install -Prelease -DskipTests $(PROFILES)
 release:
-	cd native && RUSTFLAGS="-Ctarget-cpu=native" cargo build --release
+	cd native && RUSTFLAGS="$(RUSTFLAGS) -Ctarget-cpu=native" && RUSTFLAGS=$$RUSTFLAGS cargo build --release $(FEATURES_ARG)
```

Comment:
`$$RUSTFLAGS` ? Never seen this pattern before. What does this do?

Reply:
Makefile syntax lightly different from bash from what I learned.
so in order to access enviroment variable created on fly it is needed to access is with $$

Reply:
Thanks. Learnt something new today :)